tox.ini

```diff
@@ -21,7 +21,8 @@ passenv =
     REDSHIFT_TEST_*
     PYTEST_ADDOPTS
 commands =
-  redshift: {envpython} -m pytest {posargs} tests/functional
+  {envpython} -m pytest {posargs} tests/functional -k "not tests/functional/adapter/utils"
```

Comment:
just curious, why are we separating these out?

Reply:
We hit the max on the free tier Windows box in CI like we did in `dbt-core` a few sprints ago. So I took the same approach of breaking out the largest test module count-wise. I don't like it, but it works for now, until we figure out how we want to solve for this in general.